src/metadata.rs

```diff
@@ -263,6 +257,39 @@ mod test {
 
         readme_md.write_all(readme.as_bytes()).unwrap();
 
+        let cargo_toml = cargo_toml.replace("readme.md", &readme_path);
+
+        let cargo_toml: CargoToml = toml::from_str(&cargo_toml).unwrap();
+
+        let metadata =
+            Metadata21::from_cargo_toml(&cargo_toml, &readme_md.path().parent().unwrap()).unwrap();
+
+        let actual = metadata.to_file_contents();
+
+        if actual.trim() != expected.trim() {
+            panic!(
```

Comment:
You can pass a message and formatter options as third and following parameters to assert_eq ([example](https://doc.rust-lang.org/nightly/std/macro.assert_eq.html#examples))

Reply:
Good point, that was a leftover from a slightly different implementation, I'll change that.